### PR TITLE
fix: Various issues from OCADU report

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Multiple fixes to issues identified in the OCADU report.

## Changes

- Typo fixes in documentation (side-nav other name and footer alt text)
- Switch styles overview page to use `gcds-card`
  - Allows switching the coming soon cards heading to be an `h2` to fix https://github.com/cds-snc/design-gc-conception/issues/685
 - Added `role="lisitem"` to cards on overview page to solve https://github.com/cds-snc/design-gc-conception/issues/683
 - Turn colour swatches on colour page to `role="img"` with colour description alt text to solve https://github.com/cds-snc/design-gc-conception/issues/682

### CODE REVIEW

**Comment by daine** [comment]
Looks good to me, but I have a question about:
> Turn colour swatches on colour page to role="img" with colour description alt text to solve https://github.com/cds-snc/design-gc-conception/issues/692
I understood that more to mean the images on this page https://design-system.alpha.canada.ca/en/styles/design-tokens/. Does this PR intend to close that ticket, or address a part of that ticket?

File: src/_includes/partials/token_table.njk [hunk 1]
```diff
@@ -28,14 +28,16 @@
...
+                        {%- assign colourName = colour | split: "-" -%}
+                        <div role="img" aria-label="{{ tokentable[locale][colourName[0]] }} {{ colourName[1] }}" class="color-preview" style="background-color: {{item[1].value}}; border: 1px solid {{ item[1].value | border }}"></div>
                     {%- when 'font-preview' -%}
                         {%- assign key = concatToken | append: '-' | append: item[0] %}
                         <span class="font-preview font-{{item[0]}} font-family-{{item[0]}}">{{ tokenTable.previews[key] }}</span>
                     {%- when 'spacing-preview' -%}
                         <div class="spacing-preview" style="width: {{item[1].value}}; height: {{item[1].value}};"></div>
                     {%- when 'token-name' %}
-                        --gcds-{{ concatToken | replace: ".", "-" | fixTokenName }}-{{ item[0] }}
+                        <span lang="en">--gcds-{{ concatToken | replace: ".", "-" | fixTokenName }}-{{ item[0] }}</span>
```
Comment (daine) [inline src/_includes/partials/token_table.njk:40]: What does `lang="en"` achieve here?

**Comment by ethanWallace** [comment]
> Looks good to me, but I have a question about:
> 
> > Turn colour swatches on colour page to role="img" with colour description alt text to solve [cds-snc/design-gc-conception#692](https://github.com/cds-snc/design-gc-conception/issues/692)
> > I understood that more to mean the images on this page https://design-system.alpha.canada.ca/en/styles/design-tokens/. Does this PR intend to close that ticket, or address a part of that ticket?

You're right it seems I have linked the wrong issue. Right now the issues in github/zenhub are very vague and don't have descriptions but there is an issue in the report that highlights that the colour is never mentioned only the name of the tokens, so this was to help with that. This issue might reflect the work? https://github.com/cds-snc/design-gc-conception/issues/682

File: src/_includes/partials/token_table.njk (hunk 1 above)
Comment (ethanWallace) [reply src/_includes/partials/token_table.njk:40]: Our token names are always in English (example: --gcds-text-light), the `lang="en"` is there to read all the token names properly when being read by assistive technology.

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Looks good, only request: could we make sure that all preview images for the foundation cards are the same height? They look a bit wonky right now with the different heights.

File: src/_includes/partials/token_table.njk (hunk 1 above)
Comment (daine) [reply src/_includes/partials/token_table.njk:40]: @ethanWallace thanks, that's super helpful! Love the change.

**Comment by daine** [comment]
> > Looks good to me, but I have a question about:
> > > Turn colour swatches on colour page to role="img" with colour description alt text to solve [cds-snc/design-gc-conception#692](https://github.com/cds-snc/design-gc-conception/issues/692)
> > > I understood that more to mean the images on this page https://design-system.alpha.canada.ca/en/styles/design-tokens/. Does this PR intend to close that ticket, or address a part of that ticket?
> 
> You're right it seems I have linked the wrong issue. Right now the issues in github/zenhub are very vague and don't have descriptions but there is an issue in the report that highlights that the colour is never mentioned, only the names of the tokens are, so this was to help with that. This issue might reflect the work? [cds-snc/design-gc-conception#682](https://github.com/cds-snc/design-gc-conception/issues/682)

Thanks, do you mind changing the PR summary to reflect #682 instead of #692? After that and Melanie's requests I think it should be good to go 👍

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**